config/sync/metatag.metatag_defaults.front.yml

```diff
@@ -0,0 +1,14 @@
+uuid: de62a4de-e764-4a20-95a0-4bec36b17394
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: 1noCXlegCr5HFehQRF1ViXy1jhU1jZ_sNN99a8Sj5jo
+id: front
+label: 'Front page'
+tags:
+  abstract: 'Joinup offers several services that aim to help e-Government professionals share their experience with each other. We also hope to support them to find, choose, re-use, develop and implement interoperability solutions.'
+  description: 'Joinup is a collaborative platform created by the European Commission and funded by the European Union via the Interoperability solutions for public administrations, businesses and citizens (ISA2) Programme.'
```

Comment:
https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-5795 is opened for this purpose.

Reply:
On closer inspection it turns out that these texts are not user editable and instead are hardcoded also, in the `RecommendedContentBlock`. I added a warning there, and the followup ticket has been closed. Sorry for the confusion.

Reply:
No, let's not close the other ticket. The other ticket was not to change the texts of the front page, rather than verify that the text of the front page is the text the client would like to have on the site meta tags.